Bio/Entrez/__init__.py

```diff
@@ -132,6 +139,30 @@
 api_key = None
 
 
+def format_ids(ids):
```

Comment:
Do you have strong reason to make this a public part of the API (versus a private function with a leading underscore, or even doing this inline within the ``_construct_params `` function?)

Reply:
Definitely should be private, but having it as a separate function definitely makes it easier to test.